packages/core/package.json

```diff
@@ -4069,6 +4069,7 @@
         "immutable": "^4.3.0",
         "js-yaml": "^4.1.0",
         "jsonc-parser": "^3.2.0",
+        "kill-port": "^2.0.1",
```

Comment:
This should be a devDependency, it's not necessary for the production build.

But not sure this is worth adding a dependency for. Can we choose a different port instead of 8080? Killing the "port" seems wrong. Why isn't the task that we started being killed properly?